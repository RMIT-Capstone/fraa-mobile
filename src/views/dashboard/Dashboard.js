import React from 'react';
import {View, Text} from 'react-native';
import styles from './DashboardStyle';

const Dashboard = () => {
  return (
    <View style={styles.centered}>
      <Text>This is dashboard</Text>
    </View>
  );
};

export default Dashboard;
