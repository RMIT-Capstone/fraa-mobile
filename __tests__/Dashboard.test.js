// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from '../src/views/dashboard/Dashboard';
it('renders Home.js correctly', () => {
  const tree = renderer.create(<Dashboard />).toJSON();
  expect(tree).toMatchSnapshot();
});
