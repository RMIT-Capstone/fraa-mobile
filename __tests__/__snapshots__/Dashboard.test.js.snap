// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home.js correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    This is dashboard
  </Text>
</View>
`;
